gocd/bci_repo_publish.py

```diff
@@ -18,7 +20,10 @@
 import re
 from lxml import etree as ET
 from openqa_client.client import OpenQA_Client
-from osc.core import http_GET, makeurl
+from osc.core import makeurl
+from osc.connection import http_GET
+
+_SLE_VERSION_T = Literal['15-SP3', '15-SP4', '15-SP5', '15-SP6']
```

Comment:
Does this really need to be typed?

Reply:
No it doesn't have to, but I found it quite useful, because I was wondering what the values of this variable are. Initially I thought this was a package version only to realize later that it's actually `15-SP$N`.